lecture-notes/2-vcs.md

```diff
@@ -1673,10 +1673,94 @@ TODO: Talk about interactive rebasing for squashing/prettifying.
 
 CONTENTS: Collaboration with Git
 
+You've done enough solo development. It's now time to start working with your
+groupmates on your project. In order to do that, you need to make your
+repository accessible to the others somehow.
+
+Git is pretty flexible about how repositories can be shared---after all, they
+are "just files"---but the easiest and most common way to share is via a code
+forge such as GitHub, SourceHut, or GitLab. These Git forges do a bunch of
+stuff but the most important part is hosting a copy of your `.git` folder and
+managing permissions. It's totally possible to DIY this with your own server
+and software such as Gitolite (I used to do that for years), but these forges
+*also* provide issue tracking and pull requests and other niceties.
+
+So you go ahead and make a repository on GitHub. Now you're in this weird
```

Comment:
I'd like an extra sentence explaining why making a repository on GitHub is the correct next step here, and an explicit statement that new repositories on forges start out blank.